appservice/src/utils/cpUtils.ts

```diff
@@ -85,12 +83,17 @@ export namespace cpUtils {
         });
     }
 
-    function obfuscateValue(data: string, value: string): string {
-        let astericks: string = '';
-        for (const _i of value) {
-            astericks += '*';
+    function obfuscateValue(data: string, commandOptions?: ICommandOptions): string {
```

Comment:
nit: `commandOptions?: ICommandOptions` should be `commandOptions: ICommandOptions | undefined`

The first means you can forget to pass `commandOptions`. The second forces you to pass it in every time.

Reply:
Do we want to enforce that they have to pass in commandOptions?  That means you'd always have to pass in undefined explicitly?

Reply:
This is a private function only called within this file. When would you pass `undefined`?